### ISSUE TITLE
Recording stereo unprocessed audio

### ISSUE THREAD

**Issue by maor121** [google/oboe#614]
Thank you so much for the Oboe library. It is indeed amazing with very low latency.

I have question, that may be basic: Can I record Stereo unprocessed audio with oboe?

When I try to record Stereo with Oboe. I get the same mic duplicated into two channels.
Same behaviour I observe in Android (java),  when I try to make a stereo VOICE_RECOGNITION recording (only way I found to make stereo recording so far, was Java, not voice_recognition stream. But it has AGC on it, and it's not the raw samples).

I wondered if I misuse the Oboe library or is it a limitation of the phone? (That I get same mic duplicated into two channels).

Other (maybe) relevant info:
1. Phone: Galaxy A50
2. I based my code on LifeEffect sample, with variations by Shariq from this thread:
https://github.com/google/oboe/issues/358
Basically: Blocking read mic in one thread, write to circular buffer. Another thread read from buffer and write to file.
3. AudioFormat is Float.
4.  I don't use "setDeviceId" on the input stream. Since it's stereo. I figured it need to auto detect I want both mics (I set input stream channel count to 2).

**Comment by philburk** [comment]
This mic behavior is in the underlying APIs, not Oboe.

You could try InputPreset::Unprocessed

I have seen that same behavior with stereo giving two identical signals on Pixel devices. Surprisingly, if I request 3 channels I get 3 independent mic signals.
